src/style.scss

```
@@ -35,10 +35,10 @@ header {
 main {
     padding-top: 48px;
     padding-right: 124px;
```

- [ ] Давай сделаем так
```suggestion
    padding-bottom: 48px;
    max-width: 656px;
```

---

Ничего не меняется